pages/common/jmeter.md

```diff
@@ -0,0 +1,20 @@
+# jmeter
+
+Open source java application designed for load testing functional behavior and measure performance.
+More information: <https://jmeter.apache.org/>.
```

Comment:
```suggestion
> More information: <https://jmeter.apache.org>.
```
I don't think the last slash is necessary.

Reply:
Done, please validate! ✅